product-links.txt

```diff
@@ -9,7 +9,7 @@ https://docs.camunda.io/docs/components/concepts/job-workers/
 https://docs.camunda.io/docs/guides/model-your-first-process
 https://docs.camunda.io/docs/guides/setup-client-connection-credentials
 https://docs.camunda.io/docs/guides/message-correlation
-https://docs.camunda.io/docs/apis-clients/working-with-apis-clients/
+https://docs.camunda.io/docs/apis-clients/working-with-apis-tools/
```

Comment:
Oops, I think I got this wrong.

```suggestion
https://docs.camunda.io/docs/apis-tools/working-with-apis-tools/
```

Reply:
Oh wait, I _did_ get it wrong, but in a different way -- I shouldn't touch this file at all, since it's keeping track of links from the product. I'll revert this line!